otherlibs/dynlink/Makefile

```diff
@@ -99,6 +99,7 @@ COMPILERLIBS_SOURCES=\
   parsing/builtin_attributes.ml \
   parsing/ast_mapper.ml \
   parsing/attr_helper.ml \
+  parsing/pprintast.ml \
```

Comment:
I'm not going to pretend to understand these makefiles, but I want to check that this isn't an accidental change.

Reply:
Not accidental. This file must contain all the dependencies of [[I forget which]], and the patch adds a dependency. I was uncertain about it, but Dolan (who doesn't really need to be notified about this comment) said that what I was doing was OK.